Makefile

```diff
@@ -148,6 +164,13 @@ $(OUT)/emulate.o: CFLAGS += -foptimize-sibling-calls -fomit-frame-pointer -fno-s
 # to the first target after .DEFAULT_GOAL is not set.
 .DEFAULT_GOAL :=
 
+EMCC_OBJS += $(BIN).js \
```

Comment:
It is confusing to name them object files.

Reply:
> It is confusing to name them object files.

What about `EMCC_OUTPUT`, these files are generated by `emcc`.

Reply:
Use `WEB_FILES` instead.